style.css

```diff
@@ -1,16 +1,1022 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,100&display=swap");
 
-@media (max-width: 375px) {
+@media screen and (max-width: 992px) and (min-width: 601px), (min-width: 993px) {
+  body {
```

Comment:
Sadly, the responsiveness of our elements have some issues with the desktop version (992px - 1100px) 😥

![Captura](https://user-images.githubusercontent.com/15807343/149430660-394d023b-2c10-478f-bd91-9d68557e4c49.PNG)

There are some elements that are causing this issue and the reason for this is the hard-coded widths in your CSS. If horizontally, you have previous elements that are already occupying space and you sum your hard-coded width with a new element or previous margins, the result in some cases is bigger than the actual screen size which causes the problem😥

#### Kindly be sure that your design works properly within the desktop ranges mentioned before. 

To make your life easier, you could try making a media query specifically for desktop size(992px - 1100px) until you figure out where is the problem 🧐